my suggestions

elasticdotventures opened the pull request.

elasticdotventures committed:
my suggestions

elasticdotventures committed:
blabla

elasticdotventures committed:
blender

elasticdotventures committed:
Update README.md

elasticdotventures committed:
Update README.md

elasticdotventures committed:
Update README.md

elasticdotventures committed:
Update README.md

elasticdotventures committed:
Update README.md

elasticdotventures committed:
adding chat links

elasticdotventures committed:
outlining project goals

elasticdotventures committed:
clarifying goals

elasticdotventures committed:
checkpoint

elasticdotventures committed:
checkpoint

elasticdotventures committed:
clarified significance of LSP

elasticdotventures committed:
updating as the convo continues

elasticdotventures commented:
hi @dahoo - will you accept pull requests?

elasticdotventures committed:
Update README.md

zencad notes + new tutorial

elasticdotventures committed:
does github track commits, or only pushes

elasticdotventures committed:
more links

elasticdotventures committed:
added fornjot

elasticdotventures committed:
fornjot

elasticdotventures committed:
discovery

elasticdotventures committed:
add

42sol-eu commented:
Hi @elasticdotventures I really liked your pull request - seeing that this was requested 3 days and a full year on this repository I moved it to a new repository in my control

https://github.com/42sol/19o1_awesome_list_openscad

I can give you push rights if you want.

Thanks and All Best from Germany
   Felix

elasticdotventures committed:
Update README.md

added https://chrastecky.dev/3d-printing/open-scad-configurable-calendar-3d-model

elasticdotventures committed:
Update README.md

elasticdotventures committed:
Update README.md

elasticdotventures committed:
format cleanup, added partcad (#2)
